Changelog.md

```diff
@@ -1,6 +1,28 @@
 ## Ignition Msgs 8.x
 
-### Ignition Msgs 8.0.0 (2021-xx-xx)
+### Ignition Msgs 8.0.0 (2021-09-xx)
+
+1. scene.proto: add shadow_caster_material_name
+    * [Pull request #179](https://github.com/ignitionrobotics/ign-msgs/pull/179)
+
+1. 🌐 Spherical coordinates conversion and commands
+    * [Pull request #177](https://github.com/ignitionrobotics/ign-msgs/pull/177)
+
+1. Add BoundingBox Msg
+    * [Pull request #165](https://github.com/ignitionrobotics/ign-msgs/pull/165)
+
+1. Add particle scatter ratio field to particle emitter
+    * [Pull request #162](https://github.com/ignitionrobotics/ign-msgs/pull/162)
+
+1. Add particle emitters to link.proto and add topic to particle_emitter.proto
+    * [Pull request #153](https://github.com/ignitionrobotics/ign-msgs/pull/153)
+
+1. Bump main to 8.0.0~pre1
+    * [Pull request #151](https://github.com/ignitionrobotics/ign-msgs/pull/151)
```

Comment:
this entry can be removed

Reply:
Moved into the infrastructure item in d372eec6f3bbbfb6f9280b5d5b126199b61a1971

I find it useful to list PRs like this explicitly for future reference, even if they don't directly affect the functionality.